pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -224,3 +225,14 @@ func fixPlistFileExists(agentConfig launchd.AgentConfig) error {
 	}
 	return waitForDaemonRunning()
 }
+
+func deprecationNotice() error {
+	supports, err := darwin.AtLeast("13.0.0")
+	if err != nil {
+		return err
+	}
+	if !supports {
+		logging.Warnf("This version of macOS is going to be unsupported for CRC, Please update to latest version of macOS")
```

Comment:
I would say "Please update to macOS 13 or newer"

Reply:
Done.